textile/features.textile

```diff
@@ -571,6 +571,9 @@ h3(#realtime-channels). Channels
 *** @(RTS3c1)@ If a new set of @ChannelOptions@ is supplied to @Channels#get@ that would trigger a reattachment of the channel if supplied to @RealtimeChannel#setOptions@ per "@RTL16a@":#RTL16a, it must raise an error, informing the user that they must use @RealtimeChannel#setOptions@ instead
 * @(RTS4)@ @Channels#release@ function:
 ** @(RTS4a)@ Detaches the channel and then releases the channel resource i.e. it's deleted and can then be garbage collected
+* @(RTS5)@ @Channels#getDerived@ function:
+** @(RTS5a)@ Takes @RealtimeChannel@ name and @DeriveOptions@ object as argument, to create a synthetic or derived channel. @ChannelOptions@ can be provided as an optional third argument.
```

Comment:
I think the spec should include instructions on how to synthesise the channel name from the provided `DeriveOptions`, feel free to include the regex from your implementations as a suggestion

Reply:
I think we should also just call it a "derived" channel, introducing the terminology of "synthetic" probably creates more confusion